src/zmq/zmqutil.cpp

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2014-2018 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <zmq/zmqutil.h>
+
+#include <logging.h>
+
+#include <zmq.h>
+
+void zmqError(const char *str)
```

Comment:
49e2cb78563df27ff0a5d2ad985e074ee98afcce
nanonit:
```suggestion
void zmqError(const char* str)
```